.github/workflows/system-tests.yml

```diff
@@ -11,62 +11,32 @@ on:
     - cron:  '00 04 * * 2-6'
 
 jobs:
-  system-tests-essential:
+  get-essential-scenarios:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        include:
-          - weblog-variant: express4
-          - weblog-variant: express4-typescript
-          - weblog-variant: nextjs
-    env:
-      TEST_LIBRARY: nodejs
-      WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
-      DD_API_KEY: ${{ secrets.DD_API_KEY }}
-
+    outputs:
+      scenario: ${{ steps.parse-yml.outputs.scenario }}
     steps:
       - name: Checkout system tests
         uses: actions/checkout@v4
         with:
           repository: 'DataDog/system-tests'
+      - name: Get Essential Scenarios
+        id: parse-yml
+        run: yq -o tsv .TRACER_ESSENTIAL_SCENARIOS ./scenario_groups.yml | xargs node -p "x=process.argv;x.shift();x.push('CROSSED_TRACING_LIBRARIES');'scenario='+JSON.stringify(x)" >> $GITHUB_OUTPUT
```

Comment:
Do you not need to remove first element from `process.argv` so it doesn't include the node binary?